mergin/test/test_client.py

```diff
@@ -698,3 +711,38 @@ def test_download_versions(mc):
     # try to download not-existing version
     with pytest.raises(ClientError):
         mc.download_project(project, project_dir_v3, 'v3')
+
+
+def test_paginated_project_list(mc):
+    """Test the new endpoint for projects list with pagination, ordering etc."""
+    # First remove any existing projects
+    cleanup_all(mc)
```

Comment:
Hmm can we avoid removal of all projects? The idea is to keep projects in Mergin after tests are run, so if some tests fail, it is possible to investigate the projects easily. Maybe for this test we could use a particular naming scheme of projects, and use a query string so that we match only those?

Reply:
No total cleanup anymore. The test projects for paginated endpoint got "test_paginated"-like name and it seems to work fine now.